kogito-editors-js/.gitignore

```diff
@@ -2,7 +2,6 @@
 build
 coverage
```

Comment:
TBH I started a `.gitignore` for the `boxed-expression-editor`, since it was an isolated project. So, it is perfectly fine that ore sub/super module are duplicating ignored entries. So, any suggestion would be really appreciated.
@yesamer please, feel free to propose changes (or perform directly a commit on `new-boxed-expression-master` branch), on the `.gitignore` stuff :)